### DISCUSSION TITLE
How to make Menu Choice Active?

### DISCUSSION THREAD

**Discussion by ThyLAW** [chipzoller/hugo-clarity#185] [Q&A] [answered]
Hi everyone,

By default, the Home and About menus are set to active when clicked on and brought to <domain>/Home and are highlighted. I wanted to add a menu option what will direct to <domain>/category as shown here: 
**https://neonmirrors.net/categories/technology/**

I have modified the menu and set the url just like his, but it will not actively highlight when selected. I have been messing with the files for a while now and I can't figure out how to make it work for me. 
**My site: https://lawsblog.me/**

As you can see, the "Technology" and "Other" do not highlight when selected. 

I don't really understand how Hugo works and how it generates everything which is why I am struggling here. Using Chrome's inspector I can see that the Home and About are being set under the div which has the class nav-active unlike the other ones. How can I fix this?

Thank you,

Logan

**Comment by onweru** [comment]
Hi Logan,

I bet the issue is in your config menu file. If you don't mind, I can have a look at it and guide you accordingly.

Thanks,
Weru.

**Comment by ThyLAW** [comment]
Good Afternoon,

I would be very grateful if you could look at it! Thank you very much! I have attached everything in the config -> _default directory.

[clarityconfigfiles.zip](https://github.com/chipzoller/hugo-clarity/files/6633315/clarityconfigfiles.zip)

**Comment by onweru** [comment, marked as answer]
On line 8 of  _default/menus/menu.en.toml, please add a trailing forward slash. Such that `  url = "categories/tech"` >> `  url = "categories/tech/"`.  I suggest you do the same for line 13. 

Alternatively, if you're actively tracking the theme via git, you can pull the latest version. I've updated the theme via 74975cc2ee65 such that such edits will be unnecessary.

**Comment by ThyLAW** [comment]
@onweru 

Thank you so much! I can't believe it was such an easy fix! I really appreciate your help, and thank you for the wonderful theme, I love it a lot!

**Reply by onweru** [reply]
You're most welcome. Happy blogging!
